### PR TITLE
GSP-128: Community Organization

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

### CODE REVIEW

File: rfcs/128-community-organization.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+BeyondStorage has been grown up, and we need to define our community organization to make it clear that everyone's rights and responsibilities.
+
+## Proposal
+
+I propose to add five positions for BeyondStorage, and they are separate in different projects.
+
+For now, there are two main projects:
+
+- `go-storage`: Focus on implement storage abstraction for golang, including `go-storage`, `go-service-*`, `go-endpoint`, `site`, `specs` and so on.
```
Comment (xxchan) [inline rfcs/128-community-organization.md:18]: Is `specs` under "go-storage" project? If so, I think we should state it more clearly in the repo. Btw, there's `rust` and `definitions` seems to be general.

Comment (Xuanwo) [reply rfcs/128-community-organization.md:18]: Nice catch, let's split `specs` and `site` outside from `go-storage`.

File: rfcs/128-community-organization.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- Permissions: `Write`
+
+The contributor who is recommended by Maintainer or Leader and has made an outstanding contribution to projects and needs to complete at least one feature or fix major bugs independently.
+
+### Reviewer
+
+- ID: `reviewer`
+- Permissions: `Triage`
+
+Advanced Contributor, responsible for reviewing community code, has LGTM (Looks Good To Me) approval for new submissions and can be involved in the design of subsequent projects.
```
Comment (xxchan) [inline rfcs/128-community-organization.md:47]: What does "can be involved in the design of subsequent projects" mean? I think everyone can propose a design  🤔

Comment (Xuanwo) [reply rfcs/128-community-organization.md:47]: They are copied from tikv/community, let me polish the details.

**Review by xxchan — APPROVED** [review]

**Comment by xxchan** [comment]
Ummm, GSP means "go-storage proposal", but `specs` is not only for `go-storage`... 🤔

**Comment by Xuanwo** [comment]
> Ummm, GSP means "go-storage proposal", but `specs` is not only for `go-storage`... thinking

Yep, it's wired for now, but we don't want to maintain too specs repo so far...

Maybe we can split them in the future.

**Review by Prnyself — APPROVED** [review]
LGTM

**Pull request merged**